PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/Util.java

```diff
@@ -597,4 +631,83 @@ public static String generateInstreamUriForGam(String adUnit, HashSet<AdSize> si
         return uri;
     }
 
+    private static final String GAM_VIEW_CLASS = "com.google.android.gms.ads.doubleclick.PublisherAdView";
+    private static final String GAM_CUSTOM_TEMPLATE_AD_CLASS = "com.google.android.gms.ads.formats.NativeCustomTemplateAd";
+    private static final String MOPUB_NATIVE_AD_CLASS = "com.mopub.nativeads.NativeAd";
+    private static final String INNER_HTML_SCRIPT = "document.body.innerHTML";
+
+    public static void findNative(@NonNull Object object, @NonNull PrebidNativeAdListener listener) {
```

Comment:
Maybe it should be a part of `AdViewUtils` ?

Reply:
Moved.